src/elements/RootWrapper.tsx

```diff
@@ -0,0 +1,9 @@
+import { styled } from '@material-ui/core';
+
+const RootWrapper = styled('div')(() => {
+  return {
```

Comment:
same thing as Root.  What is a RootWrapper?  Is this too small of a generalization?